third-party/snyk/snapshot.json

```diff
@@ -74,7 +74,8 @@
       {"id":"npm:highcharts:20180225","severity":"high","semver":{"vulnerable":["<6.1.0"]}}
     ],
     "jquery":[
-      {"id":"SNYK-JS-JQUERY-565129","severity":"medium","semver":{"vulnerable":[">=2.2.0 <3.5.0"]}},
+      {"id":"SNYK-JS-JQUERY-567880","severity":"medium","semver":{"vulnerable":[">=1.2.0 <3.5.0"]}},
+      {"id":"SNYK-JS-JQUERY-565129","severity":"medium","semver":{"vulnerable":[">=1.0.3 <3.5.0"]}},
```

Comment:
@snyk-bot you seriously need to make up your mind here, this vuln has gone back and forth from excluding to including jquery 2.1 (which is in our tests) 3 times now.

I know because I have to manually update the expectations when this happens! 😠

Reply:
@lirantal

Reply:
apologies on the confusion folks! looking into it with the team.

Reply:
hey folks, we updated the vulnerable version ranges to >=1.0.3 after doing another round of triage. apologies on that. See updated CVE here: https://snyk.io/vuln/SNYK-JS-JQUERY-565129

Reply:
Anytime! 🤗